sentry/src/main/java/io/sentry/SentryWrapper.java

```diff
@@ -0,0 +1,34 @@
+package io.sentry;
+
+import java.util.concurrent.Callable;
+import java.util.function.Supplier;
+
+public final class SentryWrapper {
+  public static <U> Callable<U> wrapCallable(Callable<U> callable) {
```

Comment:
could you add `final` keywords and `@Nullable` or `@NotNull` annotations?
Also, if `callable` could be null, could you add a null check?